<p>&lt;&lt;insert slide 1; Chapter Title: Title; Timecode (00:00:08). &gt;&gt;  </p>

<p><strong>&lt;&lt;Level 2&gt;&gt; Unmasking the Potential of PARP Inhibition as a Therapeutic Target in mCRPC</strong></p>

<p><a name="_Hlk508751829"></a>&lt;&lt;insert slide 2; Chapter Title: Program Agenda; Timecode (00:00:20). &gt;&gt;  </p>

<p><strong>&lt;&lt;Level 2&gt;&gt; Program Agenda</strong></p>
<p></p>
<p>&lt;&lt;insert slide 3; Chapter Title: Program Goals; Timecode (00:00:38). &gt;&gt;  </p>

<p><strong>&lt;&lt;Level 2&gt;&gt; Goals of Today's Session</strong></p>

<p>&lt;&lt;insert slide 4; Chapter Title: New Targets in Prostate Cancer: Focus on PARP Inhibition; Timecode (00:01:05). &gt;&gt;  </p>

<p><strong>&lt;&lt;Level 2&gt;&gt; New Targets in Prostate Cancer: Focus on PARP Inhibition</strong></p>

<p>&lt;&lt;insert slide 5; Chapter Title: ssDNA Repair Pathways; Timecode (00:01:15). &gt;&gt;  </p>

<p><strong>&lt;&lt;Level 2&gt;&gt; ssDNA Repair Pathways<sup>[1]</sup></strong></p>

<p>&#8226;	There are 3 primary mechanisms by which a cell fixes single-strand breaks: mismatch repair pathway (MMR), nucleotide excision repair (NER), and basic excision repair (BER)</p>

<p>&#8226;	The third of these pathways is where the <em>PARP1, 2,</em> and <em>3</em> enzymes are involved and are primarily responsible for fixing the single-strand breaks by the mechanism of BER </p>

<p>&lt;&lt;insert slide 6; Chapter Title: dsDNA Repair Pathways; Timecode (00:01:51). &gt;&gt;  </p>

<p><strong>&lt;&lt;Level 2&gt;&gt; dsDNA Repair Pathways<sup>[1,2]</sup></strong></p>

<p>&#8226;	There are 3 pathways by which DNA can fix double-strand DNA breaks and the most important and most clinically relevant is the homologous recombination pathway, which fixes double-stand DNA damage caused by ionizing radiation, as well as other double-strand DNA injury </p>

<p>&lt;&lt;insert slide 7; Timecode (00:02:31). &gt;&gt;  </p>

<p><strong>&lt;&lt;Level 2&gt;&gt; Localized PC: DNA Repair Defects<sup>[3]</sup></strong></p>

<p>&#8226;	How prevalent are these DNA repair mutations in prostate cancer? </p>

<p><tt>o	</tt>In this study the authors looked at 477 cases of localized, nonmetastatic prostate cancers and either performed whole genome sequencing on the patients, or whole exome sequencing</p>

<p><tt>o	</tt>Approximately 10% of the patients had a DNA repair mutation in their primary prostate cancer </p>

<p>&lt;&lt;insert slide 8; Chapter Title: DNA Repair Defects in mCRPC; Timecode (00:03:36). &gt;&gt;  </p>

<p><strong>&lt;&lt;Level 2&gt;&gt; DNA Repair Defects in mCRPC<sup>[4]</sup></strong></p>

<p>&#8226;	It appears that the metastatic state allows for an enrichment of the DNA repair genes, by a factor of 2-fold and maybe higher, compared with localized prostate cancer</p>

<p>&lt;&lt;insert slide 9; Timecode (00:04:22). &gt;&gt;  </p>

<p><strong>&lt;&lt;Level 2&gt;&gt; Enrichment Analysis in Primary vs Metastatic Tumors<sup>[5]</sup></strong></p>

<p>&#8226;	Which of these genes are more commonly mutated in localized prostate cancer vs advanced metastatic castration-resistant prostate cancer (mCRPC)? </p>

<p><tt>o	</tt>In this study, the sequencing effort included 918 cases, of which 335 were for metastatic and 583 were from primary tumors</p>

<p>&#9642;	The dots on the right-hand side of the graph of the dotted line represent genes more commonly altered in metastatic vs localized prostate cancers</p>

<p>&#9642;	The dots on the left are genes more commonly altered in localized disease, for example, speckle-type POZ protein (SPOP) on the left side of the dotted line, is more commonly found in localized cancer and less in metastatic cancers, whereas amplifications in androgen receptor (AR) are more prevalent than in enriched CRPC</p>

<p><a name="_Hlk515571559"></a>&lt;&lt;insert slide 10; Timecode (00:05:45). &gt;&gt;  </p>

<p><strong>&lt;&lt;Level 2&gt;&gt; Germline Mutations in PC: 1 in 10<sup>[6]</sup></strong></p>

<p>&#8226;	This trial showed a very high prevalence of germline DNA repair gene mutations in unselected patients who had metastatic prostate cancer, and of 692 individuals, 11.8% of them had a germline-inactivating pathogenic mutation in 1 of 20 DNA repair genes  </p>

<p>&lt;&lt;insert slide 11; Timecode (00:06:42). &gt;&gt;  </p>

<p><strong>&lt;&lt;Level 2&gt;&gt; Relationship Between Intraductal/Ductal Histology and Germline DNA Repair Defects<sup>[7]</sup></strong></p>

<p>&#8226;	This study illustrated the prevalence of germline DNA repair alterations enriched in patients who had variant histologies</p>

<p><tt>o	</tt>In particular, patients who had intraductal prostate cancers or ductal prostate cancers had a 40% prevalence of a germline DNA repair gene mutation, whereas 9% of patients who had classical acinar adenocarcinoma without any ductal or intraductal features had these germline mutations</p>

<p>&lt;&lt;insert slide 12; Chapter Title: The Biology of PARP Inhibition; Timecode (00:07:34). &gt;&gt;  </p>

<p><strong>&lt;&lt;Level 2&gt;&gt;</strong> <strong>The Biology of PARP Inhibition<sup>[8]</sup></strong></p>

<p>&#8226;	Poly(ADP-ribose) polymerase (PARP)1 is a key enzyme involved in fixing single-strand DNA damage via the basic excision repair pathway</p>

<p><tt>o	</tt>It binds directly to sites of single-strand damage, and once activated by the single-strand damage it uses nicotinamide adenine dinucleotide (NAD) to add these branching side chains of PARP; this is called PARylation</p>

<p><tt>o	</tt>This process results in the recruitment of a whole host of other enzymes into this protein complex; if the DNA single-strand damage is not repaired, apoptosis may ensue, and if the damage is repaired the cancer cell may survive</p>

<p>&lt;&lt;insert slide 13; Timecode (00:08:29). &gt;&gt;  </p>

<p><strong>&lt;&lt;Level 2&gt;&gt; PARP Inhibition Can Lead to Increase dsDNA Breaks<sup>[8]</sup></strong></p>

<p>&#8226;	Inactivation or inhibition of PARP can prevent recruitment of enzymes to the sites of single-stand breaks</p>

<p><tt>o	</tt>As these single-strand breaks accumulate and go unrepaired, there can be an arrest at the replication fork causing in turn a double-strand DNA break</p>

<p>&lt;&lt;insert slide 14; Timecode (00:09:05). &gt;&gt;  </p>

<p><strong>&lt;&lt;Level 2&gt;&gt; Synthetic Lethality Hypothesis<sup>[9,10]</sup></strong></p>

<p>&#8226;	This graphic shows how synthetic lethality works in the context of PARP inhibition</p>

<p><tt>o	</tt>In the first scenario there is a normal cell without a <em>BRCA</em> mutation, with normal PARP and<em> BRCA</em> function </p>

<p>&#9642;	This cell is able to repair both single-strand and double-strand DNA breaks and if a PARP inhibitor is used, PARP function is inhibited, but the normal <em>BRCA</em> function will be able to restore the DNA repair function and that cell will remain healthy and not die</p>

<p>&#8226;	In the second scenario, we have a cell that has a monoallelic or a single copy lost or inactivation of a <em>BRCA</em> gene</p>

<p><tt>o	</tt>Because these are tumor suppressor genes, both copies need to be inactivated for the <em>BRCA</em> function to be lost</p>

<p><tt>o	</tt>With a single copy or monoallelic mutation, the PARP function could be inhibited by a PARP inhibitor, but the <em>BRCA </em>function will be restored and maintained because the second allele, which is a normal allele, will recover and compensate for the faulty allele </p>

<p>&#8226;	In the third scenario in the pretreatment situation, the <em>BRCA</em> function is obliterated with both alleles lost, leaving behind only the PARP function as the only mechanism by which this cell can fix DNA damage</p>

<p><tt>o	</tt>In this scenario if a PARP inhibitor is used, the <em>BRCA</em> function is already lost because of the biallelic <em>BRCA2</em> mutation and now we are inhibiting this cell's only other mechanism that can be used to repair DNA damage and therefore this results in cell death </p>

<p>&#8226;	The key message is that it is important that both alleles or both copies of the <em>BRCA</em> gene or other genes are inactivated for PARP inhibitors to work best</p>

<p>&lt;&lt;insert slide 15; Timecode (00:11:14). &gt;&gt;  </p>

<p><strong>&lt;&lt;Level 2&gt;&gt; PARPi in <em>BRCA1/2</em> Mutation Carriers<sup>[11]</sup></strong></p>

<p>&lt;&lt;insert slide 16; Timecode (00:11:39). &gt;&gt;  </p>

<p><strong>&lt;&lt;Level 2&gt;&gt; TOPARP-A: HR Deficiency in mCRPC<sup>[12]</sup></strong></p>

<p>&#8226;	In the TOPARP-A study there was a very low response rate among patients who were biomarker-negative </p>

<p>&#8226;	There was a clear benefit in the patients who were DNA repair deficient and specifically patients with homologous repaired deficient aberrations</p>

<p>&lt;&lt;insert slide 17; Timecode (00:12:56). &gt;&gt;  </p>

<p><strong>&lt;&lt;Level 2&gt;&gt; PROfound: Olaparib vs Enzalutamide or Abiraterone Acetate in mCRPC With Prior Therapy<sup>[13</sup></strong></p>

<p>&lt;&lt;insert slide 18; Timecode (00:13:46). &gt;&gt;  </p>

<p><strong>&lt;&lt;Level 2&gt;&gt; Abiraterone in mCRPC With HRD<sup>[14]</sup></strong></p>

<p>&#8226;	The key finding in this study was that 80 patients out of the 140 in this trial had tissue-based genomics</p>

<p><tt>o	</tt>Patients who had a homologous repair gene mutation appeared to respond better to abiraterone, whether or not the PARP inhibitor was used</p>

<p><tt>o	</tt>This suggests that patients with the DNA repair mutations and the homologous repair pathway potentially do better with abiraterone</p>

<p>&lt;&lt;insert slide 19; Timecode (00:14:51). &gt;&gt;  </p>

<p><strong>&lt;&lt;Level 2&gt;&gt; Abiraterone or Enzalutamide and HRD<sup>[15,16]</sup></strong></p>

<p>&#8226;	In this study, circulating tumor DNA was used to determine the mutations and patients who had either monallelic or biallelic homologous repair (HR) gene mutations, primarily <em>BRCA2</em> or <em>ATM</em>, had inferior outcomes to first-line enzalutamide or abiraterone</p>

<p>&lt;&lt;insert slide 20; Timecode (00:15:34). &gt;&gt;  </p>

<p><strong>&lt;&lt;Level 2&gt;&gt; Pembrolizumab for MSI-H or MMR-Deficient Cancers<sup>[17]</sup></strong></p>

<p>&lt;&lt;insert slide 21; Timecode (00:16:02). &gt;&gt;  </p>

<p><strong>&lt;&lt;Level 2&gt;&gt; MMR Deficiency Across 12,019 Solid Tumors<sup>[18]</sup> </strong></p>

<p>&#8226;	Approximately 2.5% of 12,0000 tumor types were MMR-deficient, a very low percentage of prostate cancer</p>

<p>&lt;&lt;insert slide 22; Timecode (00:16:30). &gt;&gt;  </p>

<p><strong>&lt;&lt;Level 2&gt;&gt; MMR Defects in Prostatic Ductal Carcinoma<sup>[19]</sup> </strong></p>

<p>&lt;&lt;insert slide 23; Timecode (00:17:04). &gt;&gt;  </p>

<p><strong>&lt;&lt;Level 2&gt;&gt; MMR Defects and Primary Gleason Pattern 5<sup>[20]</sup></strong></p>

<p><a name="_Hlk515615139"></a>&lt;&lt;insert slide 24; Timecode (00:17:47). &gt;&gt;  </p>

<p><strong>&lt;&lt;Level 2&gt;&gt; MMR Mutations Can Cause HRD Mutations!</strong></p>

<p>&#8226;	A primary mismatch repair mutation may lead to a secondary mutation in a homologous repair gene</p>

<p>&#8226;	In this case study, the patient has a <em>MSH2</em> mutation, stop mutation E809X, and a loss of heterozygosity of the second allele, a true biallelic loss </p>

<p><tt>o	</tt>Because of this primary genetic event a number of secondary genetic events have taken place including a frame shift mutation in <em>BRCA2, ERCC4</em> and <em>5, FANCM</em> and <em>MSH6</em></p>

<p><tt>o	</tt>This patient should not receive treatment with a PARP inhibitor despite the <em>BRCA</em> mutation because this is not the driving event; the primary event in this case is the <em>MSH2</em> lesion and this patient should probably receive a programmed cell death-1 (PD-1) inhibitor</p>

<p>&lt;&lt;insert slide 25; Timecode (00:18:38). &gt;&gt;  </p>

<p><strong>&lt;&lt;Level 2&gt;&gt; Platinum Response in mCRPC With HR Deficiency<sup>[21]</sup> </strong></p>

<p>&lt;&lt;insert slide 26; Timecode (00:18:57). &gt;&gt;  </p>

<p><strong>&lt;&lt;Level 2&gt;&gt; Platinum Response in mCRPC With FANCA Deficiency<sup>[22]</sup></strong></p>

<p>&lt;&lt;insert slide 27; Timecode (00:19:17). &gt;&gt;  </p>

<p><strong>&lt;&lt;Level 2&gt;&gt; HR Deficiency and Response to Carboplatin<sup>[23]</sup></strong></p>

<p>&lt;&lt;insert slide 28; Timecode (00:19:37). &gt;&gt;  </p>

<p><strong>&lt;&lt;Level 2&gt;&gt; Conclusions</strong></p>

<p><a name="_Hlk515631133"></a>&lt;&lt;insert slide 29; Chapter Title: PARP Inhibition in mPC: What Do We Know? Timecode (00:20:42). &gt;&gt;  </p>

<p><strong>&lt;&lt;Level 2&gt;&gt; PARP Inhibition in mPC: What Do We Know?</strong></p>

<p>&lt;&lt;insert slide 30; Timecode (00:20:57). &gt;&gt;  </p>

<p><strong>&lt;&lt;Level 2&gt;&gt; Key Points to Consider<sup>[24]</sup>  </strong></p>

<p>&lt;&lt;insert slide 31; Chapter Title: First-in-Human Trials of PARP Inhibitors in Cancer; Timecode (00:22:23). &gt;&gt;  </p>

<p><strong>&lt;&lt;Level 2&gt;&gt; First-in-Human Trials of PARP Inhibitors in Cancer<sup>[1,11, 25-28]</sup></strong></p>

<p>&lt;&lt;insert slide 32; Timecode (00:23:44). &gt;&gt;  </p>

<p><strong>&lt;&lt;Level 2&gt;&gt; PARPi Sensitivity Correlates With Platinum Sensitivity<sup>[25]</sup></strong></p>

<p>&lt;&lt;insert slide 33; Timecode (00:24:19). &gt;&gt;  </p>

<p><strong>&lt;&lt;Level 2&gt;&gt; PARP Inhibitor Antitumor Activity<sup>[28,29]</sup></strong></p>

<p>&#8226;	PARP inhibition works in many different ways, in part by blocking basic excision repair; the drug blocks the PARP protein on DNA and that protein complex makes the DNA get a double-strand break; this is what kills the tumor cell, an accumulation of double-strand breaks</p>

<p>&lt;&lt;insert slide 34; Timecode (00:24:39). &gt;&gt;  </p>

<p><strong>&lt;&lt;Level 2&gt;&gt; Talazoparib<sup>[30]</sup></strong></p>

<p>&lt;&lt;insert slide 35; Timecode (00:25:45). &gt;&gt;  </p>

<p><strong>&lt;&lt;Level 2&gt;&gt; Lethal PC<sup>[4,12,31-33]</sup></strong></p>

<p>&#8226;	When cells become hormone resistant to abiraterone or enzalutamide, they develop loss of a gene called <em>RB1</em>, and the gene next to <em>RB1</em> is <em>BRCA2</em></p>

<p><tt>o	</tt>The cells concurrently lose <em>BRCA2</em> and <em>RB1</em> in at least one-third of patients, and as they get that <em>RB1</em> loss in the neuroendocrine-type phenotype, they also become more platinum sensitive, which is an important point in subclonal acquisition </p>

<p>&lt;&lt;insert slide 36; Chapter Title: Case Study; Timecode (00:26:46). &gt;&gt;  </p>

<p><strong>&lt;&lt;Level 2&gt;&gt; Patient Case</strong></p>

<p>&lt;&lt;insert slide 37; Timecode (00:27:25). &gt;&gt;  </p>

<p><strong>&lt;&lt;Level 2&gt;&gt; Patient Case (cont)</strong></p>

<p>&lt;&lt;insert slide 38; Timecode (00:28:24). &gt;&gt;  </p>

<p><strong>&lt;&lt;Level 2&gt;&gt; HRD Genomic Scar Signatures in mCRPC<sup>[34]</sup></strong></p>

<p>&lt;&lt;insert slide 39; Timecode (00:29:31). &gt;&gt;  </p>

<p><strong>&lt;&lt;Level 2&gt;&gt; Challenge of Intratumor Heterogeneity: Therapy Kills Only a Subpopulation of the Tumor Cells<sup>[33,35]</sup></strong></p>

<p>&#8226;	The aberration may not be present in a diagnostic biopsy because it can be acquired as the patient develops endocrine resistance</p>

<p>&#8226;	We should consider either rebiopsing the patient or trying to get the genomic data from the blood, maybe plasma DNA </p>

<p>&lt;&lt;insert slide 40; Timecode (00:30:29). &gt;&gt;  </p>

<p><strong>&lt;&lt;Level 2&gt;&gt; Emerging <em>RB1</em> Loss in mPC: Luminal to Basal Switch</strong></p>

<p>&#8226;	<em>RB-1</em> losses associated with repair defects in metastatic prostate cancer</p>

<p>&lt;&lt;insert slide 41; Timecode (00:35:14). &gt;&gt;  </p>

<p><strong>&lt;&lt;Level 2&gt;&gt; Subclonal and Heterogeneous <em>BRCA2</em> Loss</strong></p>

<p>&lt;&lt;insert slide 42; Timecode (00:33:20). &gt;&gt;  </p>

<p><strong>&lt;&lt;Level 2&gt;&gt; A Patient Example: Disease Plasticity and Evolution?<sup>[36]</sup></strong></p>

<p>&lt;&lt;insert slide 43; Timecode (00:33:54). &gt;&gt;  </p>

<p><strong>&lt;&lt;Level 2&gt;&gt; A Common Challenge for PARP Inhibition: DNA Repair Pathway Complexity<sup>[1]</sup></strong></p>

<p>&lt;&lt;insert slide 44; Chapter Title: TOPARP Trial Design: Olaparib in mPC; Timecode (00:34:14). &gt;&gt;  </p>

<p><strong>&lt;&lt;Level 2&gt;&gt; TOPARP Trial Design: Olaparib in mPC<sup>[12]</sup></strong></p>

<p>&#8226;	Can we use the drug olaparib to find out which cancers respond and also identify the abnormalities in the machinery that sensitize prostate cancer to the drug?</p>

<p>&lt;&lt;insert slide 45; Timecode (00:34:45). &gt;&gt;  </p>

<p><strong>&lt;&lt;Level 2&gt;&gt; TOPARP: Trial Status<sup>[12]</sup></strong></p>

<p>&lt;&lt;insert slide 46; Timecode (00:35:11). &gt;&gt;  </p>

<p><strong>&lt;&lt;Level 2&gt;&gt; TOPARP-A: Response to Olaparib in Patients With Sporadic mCRPC (Primary Endpoint)<sup>[12]</sup></strong></p>

<p>&#8226;	The longest response in this study was more than 3 years</p>

<p>&lt;&lt;insert slide 47; Timecode (00:35:51). &gt;&gt;  </p>

<p><strong>&lt;&lt;Level 2&gt;&gt; TOPARP-A: Safety<sup>[12]</sup></strong></p>

<p>&lt;&lt;insert slide 48; Chapter title: TOPARP-A: Prospective cfDNA Analyses; Timecode (00:36:05). &gt;&gt;  </p>

<p><strong>&lt;&lt;Level 2&gt;&gt; TOPARP-A: Prospective cfDNA Analyses<sup>[37]</sup></strong></p>

<p>&#8226;	The plasma can be used as a response biomarker; as the patient responds, the mutations disappear, and if the patient progresses, the mutations come back</p>

<p>&lt;&lt;insert slide 49; Timecode (00:36:40). &gt;&gt;  </p>

<p><strong>&lt;&lt;Level 2&gt;&gt; TOPARP-A cfDNA: Germline Mutation Allele Frequency in Carrier Patients After PARPi<sup>[37]</sup> </strong></p>

<p>&#8226;	If there was a germline mutation present in normal blood cells and in half the alleles, what we see is that that germline mutation still is present in the blood when the patient has responded, so the allele frequency becomes 50% when a patient has responded and is in remission </p>